src/rleCompress.js

```diff
@@ -18,7 +18,25 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  const tmp = [...source];
+  const result = [tmp[0]];
```

Comment:
`compressedParts` - as an example

Will be better to initialize variable with empty array